lib/ui/widgets/concord_tiles.dart

```diff
@@ -0,0 +1,245 @@
+import 'package:flutter/material.dart';
+import 'package:multi_image_picker/multi_image_picker.dart';
+
+class CommunityBulletinTile extends StatelessWidget {
+  final Asset image;
+  final String title;
+  final String description;
+  final Color titleColor;
+  final Color descriptionColor;
+
+  const CommunityBulletinTile({
+    Key key,
+    // TODO:  set placeholder image if one not provided
+    this.image,
+    @required this.title,
+    this.description,
+    this.titleColor,
+    this.descriptionColor
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    double radius = 4.0;
+    double height = 98;
+
+    return Container(
+      //if there is an image, the tile will be longer to accommodate the thumb nail image
```

Comment:
Style nit: Put spaces between comment delimiters and content and capitalize starts of sentences
```suggestion
      // If there is an image, the tile will be longer to accommodate the thumbnail image
```